proto/mixing_prior.proto

```diff
@@ -37,3 +37,13 @@ message LogSBPrior {
   double step_size = 2;  // TODO move?
   uint32 num_components = 3;
 }
+
+message TruncSBPrior {
+  message BetaPriors {
+    repeated BetaDistribution beta_distributions = 1;
+  }
+  oneof prior {
+    BetaPriors beta_priors = 1;
```

Comment:
I think we can also have a `DP` and `PY` prior as in Ishwaran and James, so that one does not need to specify by hand all the beta_distributions